components/occurrences/VulnerabilityOccurrenceDetails.js

```diff
@@ -28,7 +28,7 @@ const getIconRepresentation = (severity) => {
 
   if (severity === "MEDIUM") {
     iconCount = 2;
-  } else if (severity === "HIGH") {
+  } else if (severity === "HIGH" || severity === "CRITICAL") {
```

Comment:
would it make sense to have a catch all default here like `if !== "MEDIUM"` or a switch with a default?

Reply:
That is what I was trying to do by setting `iconCount = 1` right at the beginning of this function, but if you think a switch statement would be easier to read I can definitely update this! Right now, `LOW` and `SEVERITY_UNSPECIFIED` will be represented by a single flame icon.